# Command rates only apply for attitude and depth commands
cmd_roll_rate: 10 # [deg/s]
cmd_pitch_rate: 10 # [deg/s]
cmd_yaw_rate: 20 # [deg/s]
cmd_depth_rate: 0.25  # [m/s]
